README.md

```diff
@@ -25,7 +25,7 @@ Live Demo: <a href="https://quantumblacklabs.github.io/kedro-viz/" target="_blan
 
 ## Introduction
 
-Kedro-Viz is an interactive development tool for building data science pipelines with [Kedro](https://github.com/quantumblacklabs/kedro).
+Kedro-Viz is an interactive development tool for building data science pipelines with [Kedro](https://github.com/quantumblacklabs/kedro). Kedro-Viz also allows users to view and compare different experiments from the Kedro project. 
```

Comment:
suggestion: `Kedro-Viz also allows users to view and compare different experiments generated from their runs in the Kedro project.`

Reply:
Just discussed with Rashida that `Kedro-Viz also allows users to view and compare different runs in the Kedro project.` would be better.